.github/workflows/rack-box-continuous.yml

```diff
@@ -19,125 +24,80 @@ jobs:
+        python-version: 3.8
 
     - name: Package RACK CLI
-      if: steps.cache-rack-cli.outputs.cache-hit != 'true'
       run: |
         cd RACK/cli
+        python3 -m pip install --upgrade pip setuptools wheel
         pip3 wheel --wheel-dir=wheels -r requirements.txt
         pip3 wheel --wheel-dir=wheels .
         cd ${{ github.workspace }}
         tar cfz RACK/rack-box/files/rack-cli.tar.gz RACK/cli/{setup-rack.sh,wheels}
 
     - name: Package RACK ontology and data
-      if: steps.cache-rack-ontology.outputs.cache-hit != 'true'
-      run: |
-        tar cfz RACK/rack-box/files/rack.tar.gz --exclude=.git --exclude=.github --exclude=assist --exclude=cli --exclude=rack-box --exclude=tools RACK
+      run: tar cfz RACK/rack-box/files/rack.tar.gz --exclude=.git --exclude=.github --exclude=assist --exclude=cli --exclude=rack-box --exclude=tools RACK
+
+    - name: Check out RACK wiki
+      uses: actions/checkout@v2
+      with:
+        repository: ge-high-assurance/RACK.wiki
+        token: ${{ secrets.RACK_CI_PAT }}
+        path: RACK.wiki
 
     - name: Generate RACK documentation
-      if: steps.cache-rack-doc.outputs.cache-hit != 'true'
       run: |
         sudo npm install -g github-wikito-converter markdown-to-html
         gwtc -t RACK-in-a-Box RACK.wiki
         markdown -t RACK-in-a-box -s style.css RACK.wiki/Welcome.md > index.html
         mv documentation.html index.html RACK/rack-box/files
 
-    - name: Download SemTK distribution
-      if: steps.cache-semtk.outputs.cache-hit != 'true'
-      run: curl -LSs 'https://oss.sonatype.org/service/local/artifact/maven/content?r=snapshots&g=com.ge.research.semtk&a=distribution&v=2.2.2-SNAPSHOT&c=bin&e=tar.gz' -o RACK/rack-box/files/semtk.tar.gz
-
-    - name: Download style spreadsheet
-      if: steps.cache-style.outputs.cache-hit != 'true'
-      run: curl -LSs https://github.com/KrauseFx/markdown-to-html-github-style/raw/master/style.css -o RACK/rack-box/files/style.css
-
-    - name: Download systemctl script
-      if: steps.cache-systemctl.outputs.cache-hit != 'true'
-      run: curl -LSs https://github.com/gdraheim/docker-systemctl-replacement/raw/v1.5.4260/files/docker/systemctl3.py -o RACK/rack-box/files/systemctl3.py
-
     - name: Build rack-box image
       run: |
         cd RACK/rack-box
         packer build -var version=dev rack-box-docker.json
 
     - name: Login to Docker Hub
-      if: github.base_ref == 'master'
+      if: github.base_ref == 'master' && github.event_name == 'push'
       uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
 
     - name: Push rack-box image to Docker Hub
-      if: github.base_ref == 'master'
-      run: docker push interran/rack-box:dev
+      if: github.base_ref == 'master' && github.event_name == 'push'
+      run: docker push gehighassurance/rack-box:dev
```

Comment:
This allows us to check that pull requests can build rack-box images, but push rack-box images to Docker Hub only after pull requests are merged into the master branch.